test/vanilla/Expected/AcceptanceTests/XmsErrorResponse/xmserrorresponse/operations/_pet_operations.py

```diff
@@ -139,23 +178,10 @@ def do_something(
             ),
         }
         error_map.update(kwargs.pop("error_map", {}))
-        accept = "application/json"
 
-        # Construct URL
-        url = self.do_something.metadata["url"]  # type: ignore
-        path_format_arguments = {
-            "whatAction": self._serialize.url("what_action", what_action, "str"),
-        }
-        url = self._client.format_url(url, **path_format_arguments)
+        request = self._do_something_request(what_action=what_action, **kwargs)
+        kwargs.pop("content_type", None)
```

Comment:
well, why?

Reply:
@lmazuel and i talked offline. For posterity, it's bc `content-type` is request specific, so we need to make sure it's gone from the kwargs in the main operation before sending